tests/test_client.py

```diff
@@ -48,10 +48,10 @@ def expected_full_submit_url(request):
     dimension_params = []
     if request.dimensions:
         for dim in request.dimensions:
-          name = dim.name
-          min = dim.min or '*'
-          max = dim.max or '*'
-          dimension_params += [f'subset={name}({min}:{max})']
+            name = dim.name
+            min = dim.min if dim.min is not None else '*'
+            max = dim.max if dim.max is not None else '*'
```

Comment:
I'm a bit wary of this test mechanism - it feels a bit odd to test some code by running identical code in the test module. (I'm likely missing something here, though)